### ISSUE TITLE
API fora do ar

### ISSUE THREAD

**Issue by Bestfastfire** [ThiagoNelsi/dicio-api#44]
A api tá fora do ar, fica dando status 404:

`https://dicio-api-ten.vercel.app/v2/livro`

Res:

```
The deployment could not be found on Vercel.

DEPLOYMENT_NOT_FOUND
```

**Comment by gilenoms** [comment]
Alguma solução? essa API é muito útil, não abandonem o projeto.

**Comment by EduardoPazz** [comment]
🆙

**Comment by EduardoPazz** [comment]
Pessoal, como o colega @ThiagoNelsi fez deploy dessa aplicação na Vercel, creio que basta alguém criar um fork e fazer um deploy na própria conta e ver se satisfaz seus casos de uso.

No meu caso, eu precisava só obter a separação silábica. Me inspirei na ideia de fazer web scraping do dicio.com e criei uma API própria mais simples: https://github.com/EduardoPazz/syllables-api

Mas ainda sim eu poderia só ter feito fork e redeployado. Fica a dica ai pra quem tá órfão da API

**Comment by gilenoms** [comment]
> Pessoal, como o colega @ThiagoNelsi fez deploy dessa aplicação na Vercel, creio que basta alguém criar um fork e fazer um deploy na própria conta e ver se satisfaz seus casos de uso.
> 
> No meu caso, eu precisava só obter a separação silábica. Me inspirei na ideia de fazer web scraping do dicio.com e criei uma API própria mais simples: https://github.com/EduardoPazz/syllables-api
> 
> Mas ainda sim eu poderia só ter feito fork e redeployado. Fica a dica ai pra quem tá órfão da API

Verdade, eu fiz a minha API do zero e ficou ótima, não que eu seja um ninja na programação, mas é relativamente fácil fazer com python.
